roles/openldap/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 # defaults file for ansible-openldap
+openldap_initialise: false #when true it will initialise openldap again from the beginning
```

Comment:
# When enabled it will initialise the LDAP directory. If already installed, the LDAP directory
# contents will be backed up
rename openldap_init